### ISSUE TITLE
Unable to create Scrapbox page when selecting other than captured image

### ISSUE THREAD

**Issue by thinceller** [pastak/scboloo#15]
The Scrapbox page is not created if I select something other than the captured image.

Is this API specification change affecting this?
https://gyazo.com/api/docs/image

<img width="838" alt="0459592baf0065c0939f1245816b24ee" src="https://user-images.githubusercontent.com/26884355/73103635-df47d780-3f37-11ea-9623-b0dc06b50761.png">

```
uploadGyazo.js:11 POST https://upload.gyazo.com/api/upload/easy_auth 500
_callee$ @ uploadGyazo.js:11
tryCatch @ runtime.js:64
invoke @ runtime.js:355
prototype.<computed> @ runtime.js:116
step @ main.js:9474
(anonymous) @ main.js:9474
(anonymous) @ main.js:9474
(anonymous) @ uploadGyazo.js:1
exports.default @ uploadGyazo.js:1
_callee$ @ main.js:32
tryCatch @ runtime.js:64
invoke @ runtime.js:355
prototype.<computed> @ runtime.js:116
step @ main.js:8
(anonymous) @ main.js:8
Promise.then (async)
step @ main.js:8
(anonymous) @ main.js:8
Promise.then (async)
step @ main.js:8
(anonymous) @ main.js:8
(anonymous) @ main.js:8
(anonymous) @ main.js:26
(anonymous) @ MessageListener.js:18
listen @ MessageListener.js:18


main.js:9474 Uncaught (in promise) SyntaxError: Unexpected end of JSON input
Promise.then (async)
step @ main.js:9474
(anonymous) @ main.js:9474
Promise.then (async)
step @ main.js:9474
(anonymous) @ main.js:9474
(anonymous) @ main.js:9474
(anonymous) @ uploadGyazo.js:1
exports.default @ uploadGyazo.js:1
_callee$ @ main.js:32
tryCatch @ runtime.js:64
invoke @ runtime.js:355
prototype.<computed> @ runtime.js:116
step @ main.js:8
(anonymous) @ main.js:8
Promise.then (async)
step @ main.js:8
(anonymous) @ main.js:8
Promise.then (async)
step @ main.js:8
(anonymous) @ main.js:8
(anonymous) @ main.js:8
(anonymous) @ main.js:26
(anonymous) @ MessageListener.js:18
listen @ MessageListener.js:18
```

**Comment by pastak** [comment]
Thanks for your report. It is reason why image should be convert data url format. I'll fix soon 😄

**Comment by pastak** [comment]
@thinceller I have fixed and release in [v1.0.1](https://github.com/pastak/scboloo/releases/tag/v1.0.1)

**Comment by thinceller** [comment]
@pastak Thanks!
